js/scripts.js

```diff
@@ -0,0 +1,37 @@
+var prefix = "https://cors-anywhere.herokuapp.com/";
+var tweetLink = "https://twitter.com/intent/tweet?text=";
+var quoteUrl = "https://quotesondesign.com/wp-json/posts?filter[orderby]=rand&filter[posts_per_page]=1";
+
+function getQuote() {
+	$.getJSON(prefix + quoteUrl, createTweet);
+}
+
+function createTweet(input) {
+	var data = input[0];
```

Comment:
input is your response from a server. please check that you get array with at least one element